configure.ac

```diff
@@ -127,13 +127,9 @@ AX_BOOST_CHRONO()
 AS_IF([test -z "$BOOST_CHRONO_LIB"],
       [AC_MSG_ERROR(Boost.Chrono library not found. Try using --with-boost-chrono=lib)])
 
-AX_BOOST_RANDOM()
-AS_IF([test -z "$BOOST_RANDOM_LIB"],
-      [AC_MSG_ERROR(Boost.Random library not found. Try using --with-boost-random=lib)])
-
 CPPFLAGS="$BOOST_CPPFLAGS $CPPFLAGS"
 LDFLAGS="$BOOST_LDFLAGS $LDFLAGS"
-LIBS="$BOOST_CHRONO_LIB $BOOST_RANDOM_LIB $LIBS"
+LIBS="$BOOST_CHRONO_LIB $BOOST_SYSTEM_LIB $LIBS"
```

Comment:
this will conflict with another patch I have in the queue :( (to remove the dependency on boost.chrono)